CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.4.1
+
+Removes automatic `mdns_lite` setup and shifts this project to be dependency
+and documentation driven as an example of basic Nerves project setup.
+
+* Fixes
+  * Fixes a dependency ordering issue for Elixir 1.11. See https://github.com/erlang/otp/pull/2675
```

Comment:
```suggestion
  * Fixes a dependency ordering issue for Elixir 1.11. See https://github.com/nerves-project/nerves_pack/pull/42.
```

I feel slightly bad pointing to Erlang when the issue is a "won't fix" in Elixir since it breaks umbrella apps. Pointing to the Nerves Pack PR seems best.